### PR TITLE
EXP-578 Reativação do estorno de pagamento com Pix

### PR DESCRIPTION
**Pull request by diegodsgarcia**
## Contexto
O estorno de pagamento com Pix foi desativado por uma instabilidade de acordo com [essa thread](https://pagarme.slack.com/archives/C6M1XN4DQ/p1610989260019300).

Esse PR habilita novamente o botão de estorno de pagamento de uma transação

## Como testar?
Por hora o estorno só funciona em ambientes de prod, mas fica aqui o teste que eu fiz de R$ 0,01 na company do time
https://beta.dashboard.pagar.me/#/transactions/292902270

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: eb7b4357be83d202bf68d4979ecf0548877b8f0f
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: eb7b4357be83d202bf68d4979ecf0548877b8f0f
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: eb7b4357be83d202bf68d4979ecf0548877b8f0f
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: eb7b4357be83d202bf68d4979ecf0548877b8f0f
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/cockpit/src/transactions/details/isRefundable.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ const isRefundable = (transaction) => {
       ]),
       T,
     ],
-    [T, F],
+    [T],
```
Comment (vagnervst) [inline packages/cockpit/src/transactions/details/isRefundable.js:63]: só um detalhe aqui, remover o `F` faz com que a função retorne `false` e quebre os testes

**Review by vagnervst — APPROVED** [review]

**Review by ThaisRobba — APPROVED** [review]

**Pull request merged**